pkg/cmd/container/run_mount.go

```diff
@@ -159,6 +161,14 @@ func generateMountOpts(ctx context.Context, client *containerd.Client, ensuredIm
 		// When the Unmount fails, RemoveAll will incorrectly delete data from the mounted dir
 		defer os.Remove(tempDir)
 
+		// Add a lease of 1 hour to the view so that it is not garbage collected
```

Comment:
@AkihiroSuda I added a 1 hour lease. Let me know if you prefer a different value.